grant-ssh-access-forced-command.py

```diff
@@ -233,6 +234,9 @@ def is_generated_by_us(keys_file):
 
 
 def kill_all_processes(user_name: str):
+    '''try to write session before killing all processes'''
+    subprocess.call(['sudo', 'killall', '-u', user_name, '-w'])
```

Comment:
`-w` will wait forever. The manpage says:

             -w, --wait
              Wait  for  all killed processes to die.  killall checks once per
              second if any of the  killed  processes  still  exist  and  only
              returns if none are left.  Note that killall may wait forever if
              the signal was ignored, had no effect, or if the  process  stays
              in zombie state.

Thus `-w` should be avoided for SIGTERM signal (not all processes may exit on SIGTERM).